models/cardSchema.js

```diff
@@ -65,22 +50,21 @@ const userSchema = new Schema({
               type: String,
               enum: CardType,
               default: 'Task'
-        }
-    },
-  ],
-}, {
+          },
+          owner: {
+              type: Schema.Types.ObjectId,
+              ref: "users",
+              required: true,
+          },
+  },
+  {
     versionKey: false,
     timestamps: true
-});
+  }
+);
 
-userSchema.methods.setPassword = function(password) {
-  this.password = bcrypt.hashSync(password, bcrypt.genSaltSync(6));
-};
 
-userSchema.methods.validPassword = function(password) {
-  return bcrypt.compareSync(password, this.password);
-};
 
-const User = mongoose.model("Users", userSchema);
+const Card = mongoose.model("cards", cardSchema);
```

Comment:
Po destrukturyzacji na początku możemy zapisać to bez mongoose:
const Card = model("card", cardSchema);